pkg/rke/k8s_rke_system_images.go

```diff
@@ -10730,6 +10730,48 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.22.28",
 		},
+		// Enabled out of band post v2.8.9
```

Comment:
nit: Enabled in Rancher v2.9.3

Reply:
Oh nevermind, I see that we retain info of the min rancher server version it was introduced with